lib/libc/inttypes/lib_imaxdiv.c

```diff
@@ -0,0 +1,33 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <inttypes.h>
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+imaxdiv_t imaxdiv(intmax_t num, intmax_t den)
```

Comment:
where is the prototype for this api?

Reply:
https://github.com/Samsung/TizenRT/blob/90713d931de9057641dc915a38db6967e7fa5e38/os/include/inttypes.h#L214